hudi-utilities/src/main/java/org/apache/hudi/utilities/streamer/StreamSync.java

```diff
@@ -999,14 +1007,19 @@ public void runMetaSync() {
       Map<String, HoodieException> failedMetaSyncs = new HashMap<>();
       for (String impl : syncClientToolClasses) {
         Timer.Context syncContext = metrics.getMetaSyncTimerContext();
+        boolean success = false;
         try {
           SyncUtilHelpers.runHoodieMetaSync(impl.trim(), metaProps, conf, fs, cfg.targetBasePath, cfg.baseFileFormat);
+          success = true;
         } catch (HoodieMetaSyncException e) {
-          LOG.warn("SyncTool class " + impl.trim() + " failed with exception", e);
+          LOG.error("SyncTool class " + impl.trim() + " failed with exception", e);
```

Comment:
Since we don't fail the pipeline, should we keep it warn level?

Reply:
I think error is better. When I look through the logs for things not working, I assume they are logged at the error level. Later we will fail the pipeline too. It's just that we try to run the other meta syncs first.